src/java.base/share/classes/sun/net/www/MimeTable.java

```diff
@@ -367,61 +322,4 @@ int getActionCode(String action) {
         return MimeEntry.UNKNOWN;
     }
 
-    public Properties getAsProperties() {
```

Comment:
Along with dropping the methods that might be used outside of the class including the saveAsProperties method, consider making the class final. (This is just part of the implementation so it really doesn't need that declaration)
(I wonder if the Solaris implementation used the subclass or other paths).

Reply:
Marked the class `final` in dbd8bab7f6c21993ad400edb322da79b91bee80a.